tests/snippets/sort.py

```diff
@@ -0,0 +1,11 @@
+class Stable:
```

Comment:
We already have tests for sorting in ` tests/snippets/list.py ` - would be nice if they were in one place - either there or in the new file.